sormas-api/src/main/resources/strings.properties

```diff
@@ -319,7 +319,8 @@ errorSormasToSormasDeleteFromExternalSurveillanceTool = Failed to delete entitie
 errorSormasToSormasInvalidRequestMethod = Invalid HTTP verb used
 errorSormasToSormasLoadShares = Failed to load shares
 errorSormasToSormasRevokeNotPending = The request can not be revoked because it is not in pending state.
-errorQuarantineOnlyCaseAndContacts = Quarantine can only be issued for cases or contacts.
+errorQuarantineOnlyCaseAndContacts = Quarantine can only be issued for cases, contacts, event participants and travel entries.
```

Comment:
Rename this property to `errorQuarantineOnlySupportedEntities` (change it for every language so that translators don't have to do the translations again) and the next one to `errorQuarantineBulkOnlySupportedEntities`

Reply:
Done.